modules/mono/SCsub

```diff
@@ -1,121 +1,150 @@
-#!/usr/bin/env python
```

Comment:
Not sure what's wrong with this diff...

Reply:
Line endings?

Reply:
Yes, that's the file before and after your patch:
```
$ file modules/mono/SCsub 
modules/mono/SCsub: Python script, ASCII text executable
$ git apply ~/Downloads/df950b9a2950d854d0c14de966856d6296fa8dcb.patch
[...]
$ file modules/mono/SCsub 
modules/mono/SCsub: Python script, ASCII text executable, with CRLF line terminators
```